packages/miniflare/test/plugins/secret-store/index.spec.ts

```diff
@@ -0,0 +1,116 @@
+import test from "ava";
+import { Miniflare } from "miniflare";
+
+test("single secret-store", async (t) => {
+	const mf = new Miniflare({
+		verbose: true,
+		secretStores: {
+			SECRET: {
+				store_id: "test_store_id",
+				name: "secret_name",
+			},
+		},
+		modules: true,
+		script: `
+		export default {
+			async fetch(request, env, ctx) {
+				try {
+					const value = await env.SECRET.get();
+					return new Response(value);
+				} catch (e) {
+					return new Response(e.message, { status: 404 });
+				}
+			},
+		}
+		`,
+	});
+	t.teardown(() => mf.dispose());
+
+	const response1 = await mf.dispatchFetch("http://localhost");
+
+	t.is(await response1.text(), "Secret secret_name not found");
+	t.is(response1.status, 404);
+
+	const store = await mf.getSercetStoreKVNamespace("SECRET");
```

Comment:
You can now access internal kv namespace by calling `getSercetStoreKVNamespace(bindingName)` and seed the secrets.